Task_1/Manage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    public class Manage
+    {
+        public void Run()
```

Comment:
'Run' it is physical activity to get your body in good condition :).
Of course we can use such names, but try to find the best you can.